### PR TITLE
fix: api responses

### PR DESCRIPTION
**Pull request by ireznice**

### CODE REVIEW

**Comment by ireznice** [comment]
shall not be merged yet

File: web/middleware/json_params.py [hunk 1]
```diff
@@ -9,11 +9,14 @@
 
 
 async def json_params(request):
+    logger.debug(request.headers)
+    logger.debug(request.body)
     if (request.headers.get('content-type', None) == 'application/json'):
+        if (request.body == b'' or request.body == ''):
```
Comment (petrSchreiber) [inline web/middleware/json_params.py]: could be reformatted as `if (len(request.body) == 0):`

Comment (ireznice) [reply web/middleware/json_params.py]: should be fixed

File: web/lm_unit.py [hunk 2]
```diff
@@ -10,7 +11,27 @@
...
+class LMUErrorHandler(ErrorHandler):
+    def default(self, request, exception):
+
+#async def error_handler(request, exception):
+        return response.json(
+            {
+                'responses':[
+                    {
+                        'type': 'exception',
+                        'response_id': 0,
```
[the remark this answers is not in the document]
Comment (ireznice) [reply web/lm_unit.py]: it won't return anything else than this json, no other response with `response_id` set to 1 is going to be returned ever.

**Pull request merged**